index.js

```diff
@@ -1,3 +1,3 @@
 module.exports = function () {
-  return false // we've been rescued!
+  return require('false') // we've been rescued!
```

Comment:
When I tried to use npm version of "true" it failed to install, but all good with "false")